### ISSUE TITLE
Extends in tsconfig is ignored

### ISSUE THREAD

**Issue by marcusIB** [ivogabe/gulp-typescript#503]
**Expected behavior:**
Recognize 'extends' property in tsconfig.json.   

**Actual behavior:**
Does not recognize inherited file in tsconfig.json with property extends
**Your gulpfile:**

*Include your gulpfile, or only the related task (with `ts.createProject`).*
```javascript

```

**tsconfig.json**
tsconfig file
"extends": "../base",
    "include": [
        "ts/**/*",
        "typings/**/*",
        "assets/animations/**/*"
    ]

base.json file
"compilerOptions": {
        "noImplicitAny": false,
        "target": "es5",
        "declaration": true,
        "sourceMap": true
    },
    "exclude":[
        "./dist/**/*"
    ]

*Include your tsconfig, if related to this issue.*
```json

```

**Code**

*Include your TypeScript code, if necessary.*
```typescript

```

**Comment by lddubeau** [comment]
This is a duplicate of #459.

PR #494 essentially provides support for this. I've been using the code in that PR for weeks without issue.

There's some cleanup to be done but the major block is that gulp-typescript needs to have its testing infrastructure updated to test with a proper version of 2.1 (and maybe 2.2 and 2.3??). Once that is done, I can pick up where I left off, and add tests to the PR so that it the whole thing can be merged.

**Comment by ivogabe** [comment]
Closing as a duplicate of #459. I'll take a look at the tests and take another look at the PR hopefully later this week.
